tools/torch_quant/torch_quant/quantizer.py

```diff
@@ -155,68 +138,71 @@ def get_observer_types(
 
 
 class Quantizer:
-    def __init__(self, module_filter: Optional[ModuleFilter] = None,
-                 backend: Backend = Backend.REFERENCE,
-                 device: Device = Device.X86,
-                 tracer: Optional[Tracer] = None,
-                 ) -> None:
+    def __init__(
+        self,
+        module_filter: Optional[ModuleFilter] = None,
+        backend: Backend = Backend.REFERENCE,
+        device: Device = Device.X86,
+        tracer: Optional[Tracer] = None,
+        act_ob_ctr: Optional[Callable[..., Observer]] = None,
+        w_ob_ctr: Optional[Callable[..., Observer]] = None,
+        bias_ob_ctr: Optional[Callable[..., Observer]] = None,
+        qat_ob_ctr: Optional[Callable[..., Observer]] = None,
+    ) -> None:
+        if backend == Backend.FBGEMM and torch.backends.quantized.engine != 'fbgemm':
```

Comment:
If fbgemm is not available on the training platform  and the users want to deploy the trained model on a fbgemm-available platform, will this limitation too strict？

Reply:
Reasonable. Done.